x/evm/keeper/params_test.go

```diff
@@ -2,13 +2,94 @@ package keeper_test
 
 import (
 	"github.com/evmos/ethermint/x/evm/types"
+	"reflect"
 )
 
 func (suite *KeeperTestSuite) TestParams() {
 	params := suite.app.EvmKeeper.GetParams(suite.ctx)
-	suite.Require().Equal(types.DefaultParams(), params)
-	params.EvmDenom = "inj"
 	suite.app.EvmKeeper.SetParams(suite.ctx, params)
```

Comment:
nit
@Vvaradinov note that by setting the params only once before the cases, each unit test might start with different params, depending on the order of the unit tests. 

For here this is fine, but for more complicated cases you want to avoid cases being dependent on each other and do the setup below 

```
		suite.Run(tc.name, func() {

```